tests/c.test

```diff
@@ -394,3 +394,4 @@
 10014  indent_continue-8.cfg                c/sparen-indent.c
 10015  empty.cfg                            c/Issue_2845.h
 10016  Issue_3233.cfg                       c/Issue_3233.c
+10017  sp_sparen_paren.cfg                  c/double-sparen.c
```

Comment:
Please add another example with the value ignore, to test it.

Reply:
I added a test that shows that when `sp_sparen_paren=ignore`, `sp_paren_paren=add` continues to apply, which I assume is what you meant?